.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,38 +1,65 @@
----
-name: Bug report
-about: Create a report to help us improve
-title: ''
-labels: ''
-assignees: ''
+name: Bug Report
+description: Report an issue about using `main`
+labels: [kind/bug]
 
----
-
-**Describe the bug**
-A clear and concise description of what the bug is.
-
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
-
-**Expected behavior**
-A clear and concise description of what you expected to happen.
-
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
-
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
-
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser [e.g. stock browser, safari]
- - Version [e.g. 22]
-
-**Additional context**
-Add any other context about the problem here.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this bug report!
+  - type: textarea
+    id: describe-bug
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: Tell us what happened!
+      value: "When I entered 2 + 2, I got the answer 6."
+    validations:
+      required: true
+  - type: textarea
+    id: expected-bahavior
+    attributes:
+      label: What did you expect to happen?
+      description: A clear and concise description of what you expected to happen.
+      placeholder: What were you expecting to happen?
+      value: "I expected 2 + 2 to equal 4, but instead 2 + 2 equaled 6!"
+    validations:
+      required: true
+  - type: dropdown
+    id: image
+    attributes:
+      label: Image
+      description: Which specific image do you want?
+      options:
+        - Bazzite
+        - Kinoite
+        - LXQt
+        - Mate
+        - Silverblue
+        - Ubuntu
+        - Vauxite
+    validations:
+      required: true
+  - type: textarea
+    id: status
+    attributes:
+      label: Cloud Custodian version and dependency information
+      description: Please run `rpm-ostree status -v` and paste the output here.
+      render: shell
+  - type: textarea
+    id: policy
+    attributes:
+      label: Policy
+      description: Please copy and paste the policy you are trying to run if applicable. Please exclude any account/sensitive information. Your response will be automatically formatted into YAML, so code-formatting markdown is not necessary.
+      render: shell
```

Comment:
What is a policy?

Reply:
Thought I removed all this, sorry late nite last night.